spec/tasks/bulk_preserve_generic_files_spec.rb

```diff
@@ -0,0 +1,37 @@
+require 'spec_helper'
+
+describe "hydranorth:bulk_preserve_generic_files", type: :task do
+  before(:all) do
+    load_rake_environment('tasks/bulk_preserve_generic_files')
+  end
+
+  let!(:file) do
+    FactoryGirl.build(:generic_file, title: ["Foo"],) do |work|
+      work.apply_depositor_metadata('dittest@ualberta.ca')
+      work.save(validate: false)
```

Comment:
should be validate no? (its got a creator/depositor and title?). Is false thing needed?